### DISCUSSION TITLE
[2431880] 大海贼探险物语 | High Sea Saga DX {清单文件更新；开罗游戏}

### DISCUSSION THREAD

**Discussion by yuleima** [pjy612/SteamManifestCache#638] [Share]
### Steam 游戏ID

2431880

### 密钥列表

```hash
2431881;5ae964a51726dfa3bf17c04c59f3f2bca50dc7a8132dc6b90d47501160dd5d1a
```

### 清单附件

[2431880 manifests](https://github.com/pjy612/SteamManifestCache/files/15335136/2431881_6260057607116683369.zip)

### 备注

![High Sea Saga DX](https://cdn.cloudflare.steamstatic.com/steam/apps/2431880/header.jpg?t=1710925790)
找此游戏时遇到SAI.SteamAppImportPro找不到数据，AppId确认没问题，可能是未收录，于是Steam买了一份，分享key和mainfest。

email : 17914393@qq.com
code : 995A8FD23A4D7188

**Comment by pjy612** [comment]
感谢分享

**Comment by yuleima** [comment]
20240516 清单文件已更新。
虽然steam上开罗游戏的更新大多是为了更新而更新，对游戏内容几乎没影响……还是同步一下吧，避免旧清单不可用的问题。
